.github/workflows/interface-unit-tests.yml

```diff
@@ -219,7 +271,9 @@ jobs:
       - setup-ci-load
       - determine_runner
       - default-dependency-versions
+      - disable-fail-fast-wae
     strategy:
+      fail-fast: ${{ contains(fromJSON(needs.disable-fail-fast-wae.outputs.matrix-fail-fast),  'default') }}
```

Comment:
```suggestion
      fail-fast: ${{ contains(fromJSON(needs.disable-fail-fast-wae.outputs.fail_fast),  'default') }}
```

Reply:
^ This suggestion but for all instances of this

Reply:
Done